tests/unit/Pagination.ts

```diff
@@ -145,8 +145,6 @@ registerSuite({
 							onPageRequest: widget.listener
 						}),
 						v('span', {
-							afterCreate: widget.listener,
```

Comment:
These will need to be added back in with the latest widget-core unfortunately (released earlier today)